src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/_notes.html.twig

```diff
@@ -0,0 +1,8 @@
+{% if order.notes is defined and order.notes is not null %}
```

Comment:
I think ``order.notes`` is always defined, as it's ``Order``'s property, so ``is not null`` should be sufficient, wdyt?